{"sections":[],"abstract":[{"type":"text","text":"Inherited from "},{"type":"codeVoice","code":"LocalizedError.errorDescription"},{"text":".","type":"text"}],"primaryContentSections":[{"kind":"declarations","declarations":[{"platforms":["macOS"],"languages":["swift"],"tokens":[{"kind":"keyword","text":"var"},{"kind":"text","text":" "},{"kind":"identifier","text":"errorDescription"},{"kind":"text","text":": "},{"kind":"typeIdentifier","preciseIdentifier":"s:SS","text":"String"},{"kind":"text","text":"? { "},{"kind":"keyword","text":"get"},{"kind":"text","text":" }"}]}]}],"variants":[{"traits":[{"interfaceLanguage":"swift"}],"paths":["\/documentation\/postgresclientorm\/tableobjecterror\/errordescription"]}],"identifier":{"interfaceLanguage":"swift","url":"doc:\/\/PostgresClientORM\/documentation\/PostgresClientORM\/TableObjectError\/errorDescription"},"metadata":{"role":"symbol","title":"errorDescription","externalID":"s:17PostgresClientORM16TableObjectErrorO16errorDescriptionSSSgvp","roleHeading":"Instance Property","modules":[{"name":"PostgresClientORM"}],"symbolKind":"property","fragments":[{"kind":"keyword","text":"var"},{"text":" ","kind":"text"},{"text":"errorDescription","kind":"identifier"},{"kind":"text","text":": "},{"preciseIdentifier":"s:SS","text":"String","kind":"typeIdentifier"},{"text":"?","kind":"text"}]},"hierarchy":{"paths":[["doc:\/\/PostgresClientORM\/documentation\/PostgresClientORM","doc:\/\/PostgresClientORM\/documentation\/PostgresClientORM\/TableObjectError"]]},"kind":"symbol","schemaVersion":{"minor":3,"patch":0,"major":0},"references":{"doc://PostgresClientORM/documentation/PostgresClientORM/TableObjectError/errorDescription":{"identifier":"doc:\/\/PostgresClientORM\/documentation\/PostgresClientORM\/TableObjectError\/errorDescription","kind":"symbol","fragments":[{"text":"var","kind":"keyword"},{"text":" ","kind":"text"},{"text":"errorDescription","kind":"identifier"},{"kind":"text","text":": "},{"preciseIdentifier":"s:SS","text":"String","kind":"typeIdentifier"},{"text":"?","kind":"text"}],"type":"topic","role":"symbol","title":"errorDescription","url":"\/documentation\/postgresclientorm\/tableobjecterror\/errordescription","abstract":[]},"doc://PostgresClientORM/documentation/PostgresClientORM":{"identifier":"doc:\/\/PostgresClientORM\/documentation\/PostgresClientORM","title":"PostgresClientORM","role":"collection","url":"\/documentation\/postgresclientorm","type":"topic","kind":"symbol","abstract":[]},"doc://PostgresClientORM/documentation/PostgresClientORM/TableObjectError":{"title":"TableObjectError","abstract":[],"url":"\/documentation\/postgresclientorm\/tableobjecterror","identifier":"doc:\/\/PostgresClientORM\/documentation\/PostgresClientORM\/TableObjectError","navigatorTitle":[{"kind":"identifier","text":"TableObjectError"}],"role":"symbol","fragments":[{"kind":"keyword","text":"enum"},{"kind":"text","text":" "},{"text":"TableObjectError","kind":"identifier"}],"kind":"symbol","type":"topic"}}}